Dockerfile.template.erb

```diff
@@ -150,7 +150,15 @@ RUN apt-get update \
                   '*-dev' \
  && rm -rf /var/lib/apt/lists/* \
 <% end %>
- && rm -rf /tmp/* /var/tmp/* /usr/lib/ruby/gems/*/cache/*.gem /usr/lib/ruby/gems/3.*/gems/fluentd-*/test
```

Comment:
As this template is applied to not only alpine, it may be better to do in conditional section.

```
<% if is_alpine %>
  ...
  &&  rm -rf \

 (remove under /usr/lib/ruby...)
  ...
<% else %>
   ...
  && rm -rf \
 ...
 (remove under /usr/local/bundle/ruby/...)
<% end %>
```

Reply:
But this is not Alpine specific. I only wrote the size difference for Alpine, but the other images are reduced as well.

Reply:
It seems that based version of template.erb (original) is already Alpine specific one about removing unused files. 

> But this is not Alpine specific. I only wrote the size difference for Alpine, but the other images are reduced as well.

It is true about /tmp/* and /var/tmp/*, but not true for the rest of targets.

So, it is not appropriate to apply Alpine specific fix in common section.

I mean:

Before:

```
<% end %>
 && rm -rf /tmp/* /var/tmp/* /usr/lib/ruby/gems/*/cache/*.gem /usr/lib/ruby/gems/3.*/gems/fluentd-*/test
```

After:

```diff
- && rm -rf /tmp/* /var/tmp/* /usr/lib/ruby/gems/*/cache/*.gem /usr/lib/ruby/gems/3.*/gems/fluentd-*/test
+ && rm -rf /tmp/* /var/tmp/* \
+<% if is_alpine %>
+ /usr/lib/ruby/gems/*/cache/*.gem \
+ && find /usr/lib/ruby/gems -type d -name test -or -name ext -or -name spec -or -name benchmark | xargs rm -rf 
+<% else %>
+ /usr/local/lib/ruby/gems/3.*/cache/*.gem \
+ && find /usr/local/bundle/gems -type d -name test -or -name ext -or -name spec -or -name benchmark | xargs rm -rf
+<% end %>
```

Reply:
Oops, I missed this comment. 

I'll fix it, thank you.

Reply:
Done